pkg/quota/evaluator/core/pods.go

```diff
@@ -337,6 +366,26 @@ func isTerminating(pod *api.Pod) bool {
 	return false
 }
 
+func doesPriorityClassExist(pod *api.Pod) bool {
+	if pod.Spec.PriorityClassName != "" {
+		return true
+	}
+	return false
+}
+
+func isPriorityClassIn(pod *api.Pod, scope string) bool {
+	if pod.Spec.PriorityClassName == "" {
+		return false
+	}
+	classes := strings.Split(scope, ":")[1:]
```

Comment:
Why PriorityClassNames are also splitted by ":"?
Should they be splited by "," ?

Reply:
agree ^

Reply:
+1 and do they start with ":"?

Reply:
@bsalamat like this for example `PriorityClassNameIn:another-priorityclass-name:cluster-services`

We cannot use `,` as a delimiter between class names on cli because `,` is delimiter between scopes.
PTAL at [@derekwaynecarr  comment](https://github.com/kubernetes/kubernetes/pull/57963#discussion_r182766953)
>$ kubectl create quota myquota --hard=pods=10 --scopes=BestEffort,Terminating,PriorityClassNameIn:foo,bar

Reply:
Thanks. Do we have any other feature that gets its list like this (i.e., with `:` delimiters)? If not, we should probably use what other features use.

Reply:
@bsalamat I searched through [Kubectl cli reference](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands), but could not find an existing command which expresses a nested list i.e similar to "list of priority classes within list of scopes".

some commands from expressing lists are there are the below ones:
> kubectl create quota my-quota --hard=cpu=1,memory=1G,pods=2,services=3,replicationcontrollers=2,resourcequotas=1,secrets=5,persistentvolumeclaims=10
> kubectl create clusterrole foo --verb=get,list,watch --resource=rs.extensions
> kubectl create clusterrole foo --verb=get,list,watch --resource=pods,pods/status

But none seems to be expressing nested list.
Wondering if you have any suggestion to make on top of your mind.

/cc @derekwaynecarr

Reply:
ping @bsalamat @derekwaynecarr

Reply:
yeah dont know if there is any better way here that includes ',' to represent nested list.

Reply:
@bsalamat How do we represent case like this, `scopes=PriorityClassNameIn=class1,class2,class3,BestEffort,Terminating` ?

Reply:
For the record, we chatted offline and decided to go with:
`scopes=PriorityClassNameIn=class1;class2;class3,BestEffort,Terminating`